thirdeye/thirdeye-frontend/app/pods/components/rootcause-settings/template.hbs

```diff
@@ -6,4 +6,99 @@ baselineRangeStart: {{input value=baselineRangeStart}} |
 baselineRangeEnd: {{input value=baselineRangeEnd}} |
 analysisRangeStart: {{input value=analysisRangeStart}} |
 analysisRangeEnd: {{input value=analysisRangeEnd}} |
```

Comment:
please feel free to remove any of these old fields

Reply:
will keep it until we have query params. I like them for debugging purpose :)